### DISCUSSION TITLE
Is dockge file system aware of the local folder of the compose file?

### DISCUSSION THREAD

**Discussion by ksurl** [louislam/dockge#26] [Ask for Help] [answered]
I tried migrating a few stacks to dockge to test it out. I thought why not use uptime-kuma. when starting it up from dockge, it prompts to setup a new admin user/password like a brand new install. my volume for the app data is same as before ./data:/app/data (data is in the same folder as the compose file). when I removed that stack and then started it via docker compose cli, it brought back my existing instance with all the old monitors configured. 

is this a bug, not implemented yet or not supported?

**Comment by louislam** [comment]
Are you using the default stacks directory `/opt/stacks/` ?

**Reply by ksurl** [reply]
I set the environment variable to /compose. And it does detect all the stacks when i click scan

**Reply by louislam** [reply, marked as answer]
If you changed the default stacks path, make sure the volume bind also are the same. 

Like this:
```
    volumes:
      - /compose:/compose  # Must be the same
    environment:
      - DOCKGE_STACKS_DIR=/compose
```

**Reply by ksurl** [reply]
it has to be /compose:/compose? that may be the issue, since my host path is not /compose.
I'll try it with a symlink for /opt/stacks instead

**Reply by ksurl** [reply]
confirmed working after i made the volume mount match on host and container. I used a symlink and it worked fine.

**Reply by adizag** [reply]
@louislam is there a way to use the exact same file for several stacks?

**Comment by ksurl** [comment]
> @louislam is there a way to use the exact same file for several stacks?

open a new discussion. but the answer is no because that's not how compose works. if you want multiple containers in one compose it will show as 1 stack. if you want multiple stacks, you need multiple compose files in multiple folders
